src/scss/_main.scss

```diff
@@ -505,10 +505,17 @@
 
 	&:before,
 	&:after {
-		// scss-lint:disable DuplicateProperty
-		content: ' ';
-		display: none;
-		display: table\9; // Show only to IE<=9
+		content: '';
+		display: table;
+		//we only want display: table (and therefore the clear:both) to apply if flexbox is not supported
+		& {
+			// Prevents autoprefixer from outputting display: -webkit-box;, which is buggy
+			// NOTE - needs to be in its own block, as the autoprefixer: off comment applies to the whole block
+			/*autoprefixer: off*/
```

Comment:
When using flexbox, it used to be display none, why is it now saying display flex?